spec/requests/invitations_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "rails_helper"
+
+RSpec.describe "Invitations", type: :request do
+  let(:user) { create(:user) }
+
+  describe "Accept invitation" do
+    it "renders normal response even if site config is private" do
+      allow(SiteConfig).to receive(:public).and_return(false)
+      get "/users/invitation/accept?invitation_token=blahblahblahblah"
+      # This is a fake token, so the only thing we're testing for here is
+      # that we *do not* land on the "registrations" page which shouldn't
+      # interrupt the request, even for private forems. 
+      expect(response.body).not_to include("registration__actions")
```

Comment:
**(non-blocking)** I think doing a check like `expect(request.path).to eq("/users/invitations/accept"` is less brittle than checking for HTML on the page.

Reply:
Yeah, but the path wouldn't redirect. The only thing we do is render a different view. There probably _are_ different tests we could do but nothing is immediately jumping out to me.